hvac/api/auth_methods/aws.py

```diff
@@ -128,7 +128,7 @@ def delete_config(self, mount_point=AWS_DEFAULT_MOUNT_POINT):
         return self._adapter.delete(url=api_path)
 
     def configure_identity_integration(
-        self, iam_alias=None, ec2_alias=None, mount_point=AWS_DEFAULT_MOUNT_POINT
+        self, iam_alias=None,  iam_metadata=None, ec2_alias=None, ec2_metadata=None, mount_point=AWS_DEFAULT_MOUNT_POINT
```

Comment:
Let's move the new parameters to the end of the parameter list, that way we don't break anyone using positional arguments.

Reply:
Good call, took care of that. Do you resolve these threads when you re-review or should I be doing that?